src/sickle.ts

```diff
@@ -120,6 +126,7 @@ class Annotator {
     return {
       output: this.output.join(''),
       externs: null,
+      diagnostics: this.diagnostics,
```

Comment:
I think you want to reset the diagnostics on each annotate call, similar to output. My mental model is that a single Annotator can be called multiple times.